client/src/reducers/differential.js

```diff
@@ -47,14 +47,6 @@ const Differential = (
         celllist1: null,
         celllist2: null,
       };
-    case "reset subset":
```

Comment:
Can you explain the goal and side-effects of this change?   My guess is that the intent is to not delete the gene list created by the diffexp request) when doing a subset/reset.

However, this change has at least one other side effect -- upon a "subset to selection", the cell lists will now potentially contain cells not visible.  I'm not sure what side effects that will  have, but it violates our past assumptions about the cell lists always being a subset of the currently viewable cells (aka "world").  I am suspicious it will lead to future bugs.

Why not leave the previous celllist clear code, but remove the diffExp clear?  That also has potential for confusion, in that the statistics (eg, p-val, ...) will no longer represent the diffexp result of the current cell sets, but at least it won't break the world/universe constraints.

TL;DR - this change needs some thought, as it violates some constraints I thought we considered important.

Reply:
Agreed, this should be discussed fully in an issue such as `reconsider diffexp now that we have gene sets and the results of diffexp are displayed in a gene set`. I'm not tied to diffexp functionality in this PR, such as changing the number of genes returned, but @signechambers1 and I should work together I think to capture where we're headed.

Reply:
OK, seems reasonable. So are you going to revert all the diffexp stuff in this PR, or do we need to resolve the design issues before landing it?

Reply:
I'd rather revert. cc @signechambers1

Reply:
fully concur (ditto for the number of results). We can turn this around in another PR quickly, if we know what behavior we want.